### ISSUE TITLE
Image file not found

### ISSUE THREAD

**Issue by josenimo** [labsyspharm/minerva-author#32]
Dear Minerva developer,

I am Jose, a biologist delving into imaging. I have tried to run minerva with the executable windows file, and by installing it folowing your instructions.  Upon trying to import the any picture it just states "Image not found ... "
While trying to install via github/conda, I ran into some installing errors, some packages could not be installed by running the parameters.yml, pip crashes for the last few. I installed those packages separately. Also, upon running the pytest in the minerva-author folder it also ran into some errors, I have added the report as a text file
[pytest minerva.txt](https://github.com/labsyspharm/minerva-author/files/7979676/pytest.minerva.txt)

Thank you in advance for your help!
Best,
Jose

**Comment by thejohnhoffer** [comment]
Hi @josenimo. Thank you so much for your interest in Minerva. From the `pytest` report, I can see the solution would be to upgrade the python version installed on your system to `python3`. One way would be to install python3 [through Anaconda](https://docs.anaconda.com/anaconda/install/windows/).

For the "Image not found" issue, could I ask what is the format of your image? Minerva Author supports certain [OME-TIFF files](https://docs.openmicroscopy.org/ome-model/6.2.3/) designed for multichannel high resolution images. There are a number of formats that can be converted to OME-TIFF using a separate tool called [Bio-Formats](https://www.openmicroscopy.org/bio-formats/).

Thanks again for your interest in our software, and  let me know if I haven't yet answered your questions.

**Comment by josenimo** [comment]
Hi @thejohnhoffer. This is interesting, I installed everything following your instructions, including the python 3.8, which is the version I see when I type `python -V` in the command line in the conda environment. I have no idea why Python 2.7 is used specifically with the pytest, to be honest I have never used a pytest before. 

The format is OME-TIFF, I am pasting a download link to this post with the said image. Perhaps there is something different with it. If it is of any help, I already ran MCMICRO with it.

https://filetransfer.mdc-berlin.de/?u=Papz4RyT&p=vGvMvQjw
